pulp_smash/tests/pulp3/utils.py

```diff
@@ -286,3 +286,35 @@ def get_repo_versions(repo):
         [version['_href'] for version in versions],
         key=lambda url: int(urlsplit(url).path.split('/')[-2])
     )
+
+
+def get_artifact_paths(repo, version_href=None):
+    """Return the paths of artifacts present in a given repository version.
+
+    :param repo: A dict of information about the repository.
+    :param version_href: The repository version to read.
+    :returns: A set with the paths of units present in a given repository.
+    """
+    return {
+        content_unit['artifact']  # file path and name
+        for content_unit in get_content(repo, version_href)['results']
+    }
+
+
+def delete_repo_version(repo, version_href=None):
+    """Delete a given repository version.
+
+    :param repo: A dict of information about the repository.
+    :param version_href: The repository version that should be
+        deleted.
+    :returns: A tuple of information about the task created when deleting
+        a given repository version.
```

Comment:
This method doesn't necessarily return a single task. It returns a tuple of tasks. Maybe this could read:

> A tuple. The tasks spawned by Pulp.